run/events-pubsub/app.js

```diff
@@ -4,6 +4,7 @@
 
 // [START run_events_pubsub_handler]
 const express = require('express');
+const {toMessagePublishedEvent} = require('@google/events/cloud/pubsub/v1/MessagePublishedData');
```

Comment:
Nit - I don't imagine someone can do something like this?

`const {toMessagePublishedEvent} = require('@google/events/cloud/pubsub/v1').MessagePublishedData;`

(I ask because this format makes reverse engineering/"exploring" the "type definition universe" a bit easier, via Node's built-in debugging tools.)

Reply:
Yup, we can do that. Going to hold off for right now though.